.github/release-please/config.json

```diff
@@ -9,6 +9,9 @@
   ],
   "packages": {
     "packages/core/js-client": {},
+    "packages/core/js-client-isomorphic": {
+      "release-as": "0.1.0"
```

Comment:
don't really know about how release please is configured. What does this setting mean? will it be removed in the future? looks a bit concerning to me that there is an exact number here

Reply:
`release-as` will make this package version set to 0.1.0 permanently.

If you want to set the first version to 0.1.0 you need to add it to .github/release-please/manifest.json

Reply:
Yes, probably we can make it different way. I will release 0.1.0 manually and set that to manifest instead of `release-as`